database/migrations/2018_08_31_203825_create_meetups_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateMeetupsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('meetups', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            $table->integer('user_id');
```

Comment:
This column doesn't seem to have a relation

Reply:
i'm not done with this yet, but thanks for pointing it out ^^